packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -140,7 +136,6 @@ export class PrometheusExporter implements MetricExporter {
       metric.inc(
         labelValues,
         point.value as Sum,
-        hrTimeToMilliseconds(point.timestamp)
```

Comment:
why is this not needed anymore ?

Reply:
Version 12 of the prom client removed the timestamp https://github.com/siimon/prom-client/pull/333